examples/barotropicqg_betaforced.jl

```diff
@@ -55,7 +55,6 @@ kf, dkf = 14.0, 1.5     # forcing wavenumber and width of forcing ring in wavenu
 
 gr  = TwoDGrid(nx, Lx)
 
-x, y = gridpoints(gr)
 Kr = [ gr.kr[i] for i=1:gr.nkr, j=1:gr.nl]
 
 forcingcovariancespectrum = @. exp(-(sqrt(gr.Krsq)-kf)^2/(2*dkf^2))
```

Comment:
This line is hard to read and somewhat inscrutable. I might infer that `kf` is a forcing wavenumber... but maybe writing `forcing_wavenumber` would make this unambiguous. A comment that explains the point of this could help too. `dkf` could be named `forcing_bandwidth`.